tests/edge-cases/logand-01.txt

```diff
@@ -0,0 +1,2 @@
+brun '(logand)'
+-1
```

Comment:
@aqk this behavior is documented, right?

Reply:
Yes, it's in the table of outputs with zero arguments, and in the arithmetic unity table, but there it's described as "all ones", so I'll improve that.